frontend/src/app/modules/auth/register-form/register-form.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { passwordMatchValidator } from '../validators/customValidator';
+
+@Component({
+  selector: 'app-register-form',
+  templateUrl: './register-form.component.html',
+  styleUrls: ['./register-form.component.scss']
+})
+export class RegisterFormComponent {
+
+  public registerForm: FormGroup;
+
+  public hidePass: Boolean = true;
```

Comment:
```suggestion
  public hidePass = true;
```
You can omit specifying type if you initializing it with a value